landingai/io.py

```diff
@@ -101,3 +100,82 @@ def sample_images_from_video(
         frame_count += 1
     cap.release()
     return output
+
+
+def read_from_notebook_webcam() -> Callable[[], str]:
+    # Define function to acquire images from the web browser
+    from IPython.display import display, Javascript
```

Comment:
our Python SDK doesn’t install this dependency by default.
could be a surprise to the user if they accidentally runs it. 
shall we add an error handling here? i.e.

try:
   from IPython.display import display, Javascript
except:
   _LOGGER.error("This function is only intended for notebook usage. Please install the package with ....."

Reply:
Good catch!... I moved it inside the try which pourpose is exaclty to detect the context